src/app/Main/Main-Layout.component.tsx

```diff
@@ -12,29 +12,10 @@ const wrapperStyle = css`
     box-sizing: border-box;
 `;
 const bodyStyle = css`
-    width: 100%;
-    flex: 1 0 auto;
+    flex: 1;
```

Comment:
I recommended to use the long form `` 1 1 0``. E.g. i had to look in to the docs to check what the short form stands for.

Reply:
Done.